src/Components/Home/ArticleList.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import axios from "axios";
+import ArticleCard from "../ArticleCard";
+
+function ArticleList() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("https://nc-news-app-ftk2.onrender.com/api/articles")
+      .then((res) => {
+        setArticles(res.data.articles[0]);
+      });
+  }, []);
+
+  return (
+    <div className="w-8/12 mx-auto">
+      <ul className="flex flex-col gap-5">
+        {articles && articles.length > 0 ? (
+          articles.map((article) => (
+            <li key={article.article_id}>
+              <ArticleCard article={article} />
+            </li>
+          ))
+        ) : (
+          <li>No articles available</li>
```

Comment:
It would be nicer if this were delivered as a response from the BE (sending an error message in the instance, for example, all articles had been deleted). There may well be functionality already setup on your BE to deal with this. Consider this as part of the error handling ticket later on.